documentation/bin/resolveTransitiveCalls.groovy

```diff
@@ -0,0 +1,179 @@
+import groovy.json.JsonSlurper
+
+def cli = new CliBuilder(
+    usage: 'groovy createDocu [<options>]',
+    header: 'Options:',
+    footer: 'Copyright: SAP SE')
+
+cli.with {
+    i longOpt: 'in', args: 1, argName: 'file', 'The file containing the mapping as created by the unit tests..'
+    o longOpt: 'out', args: 1, argName: 'file', 'The file containing the condenced mappings.'
+    h longOpt: 'help', 'Prints this help.'
+}
+
+def options = cli.parse(args)
+
+if(options.h) {
+    System.err << "Printing help.\n"
+    cli.usage()
+    return
+}
+
+if(! options.i) {
+    System.err << "No input file"
+    cli.usage()
+    return
+}
+if(! options.o) {
+    System.err << "No output file"
+    cli.usage()
+    return
+}
+
+def steps = new JsonSlurper().parseText(new File(options.i).text)
+
+def piperSteps = steps.piperSteps
+def calls = steps.calls
+
+def _calls = [:]
```

Comment:
how is `_calls` different than `calls`? Can we have a better name here?

Reply:
done